src/current/_includes/common/license-expiration-messages.md

```diff
@@ -0,0 +1,9 @@
+Depending on the status of your cluster's [license]({% link {{ page.version.version }}/licensing-faqs.md %}), the following messages may be displayed in the DB Console:
+
+- If the license will expire soon, the message is: _This cluster will require a license key by ${DATE} or the cluster will be throttled._
+- If the license is no longer valid, the message is: _Your license key expired on ${DATE} and the cluster was throttled. Please add a license key to continue using this cluster._
```

Comment:
This message is technically displayed after a grace period and not immediately after a license expires. I don't think we need to advise users about the grace periods, but I'm wondering if we should say "if your cluster was throttled because of license validity issues..." or something along those lines.

Reply:
There is a separate message displayed if the cluster was throttled because no license key was ever added: "This cluster was throttled because it requires a license key. Please add a license key to continue using this cluster. Learn more"

Reply:
> This message is technically displayed after a grace period and not immediately after a license expires. I don't think we need to advise users about the grace periods, but I'm wondering if we should say "if your cluster was throttled because of license validity issues..." or something along those lines.

I have updated this to the following:

- If your cluster was throttled due to an invalid license, the message is: _Your license key expired on ${DATE} and the cluster was throttled. Please add a license key to continue using this cluster._

> (https://github.com/cockroachdb/docs/pull/19002#discussion_r1820865979)
There is a separate message displayed if the cluster was throttled because no license key was ever added: "This cluster was throttled because it requires a license key. Please add a license key to continue using this cluster. Learn more"

Oh good, I missed that case, thanks - added the following to the list:

- If your cluster was throttled because a valid license key was never added, the message is: _This cluster was throttled because it requires a license key. Please add a license key to continue using this cluster._

Also re-ordered the bullets so the cases are in the following order which is hopefully easier to understand:

- No valid key ever entered, grace ended
- Invalid key, still in grace
- Invalid key, out of grace (throttled)
- No telemetry (throttled)

Please let me know if these updates are right and/or if i missed anything from your comments